src/NServiceBus.AwsLambda.SQS/SQSEventExtensions.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace NServiceBus.AwsLambda.SQS
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using Amazon.SQS.Model;
+    using static Amazon.Lambda.SQSEvents.SQSEvent;
+
+    static class SQSEventExtensions
+    {
+        public static Message ToMessage(this SQSMessage source)
+        {
+            var target = new Message();
+
+            target.SetIfNotNull(source.MessageId, static (t, value) => t.MessageId = value);
```

Comment:
wouldn't `target.MessageId = source.MessageId ?? target.MessageId` be equivalent?